.github/workflows/check-catalyst.yaml

```diff
@@ -533,7 +532,7 @@ jobs:
             mv runtime/build/coverage.info coverage-${{ github.job }}.info
 
       - name: Build Runtime test suite with both Lightning and Lightning-Kokkos simulators
-        if: ${{ matrix.backend == 'lightning-kokkos' }}
+        if: false
```

Comment:
Does it need to be `false` even though the compiler is set to gcc?

Reply:
```suggestion
        if: ${{ matrix.backend == 'lightning-kokkos' }}
```

Reply:
Let's find out